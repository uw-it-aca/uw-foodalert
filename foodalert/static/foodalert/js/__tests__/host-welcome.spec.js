"use strict";
import Vue from 'vue';
import BootstrapVue from 'bootstrap-vue'
Vue.use(BootstrapVue);

import {mount} from '@vue/test-utils';
import WelcomePage from '../pages/host/welcome.vue';

describe('Welcome Page', function () {
        test('renders correctly', function () {
            var wrapper = mount(WelcomePage, {
            });
            expect(wrapper.element).toMatchSnapshot();
        });
})