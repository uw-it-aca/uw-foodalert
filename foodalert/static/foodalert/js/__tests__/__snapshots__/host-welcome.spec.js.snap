// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Page renders correctly 1`] = `
<div>
  <div
    class="page"
  >
    <div
      id="standard-notification"
    >
      <div
        class="collapse"
        id="notif-container"
        style="display: none;"
      >
        <div
          class="text-center py-3 container-fluid"
        >
          <div
            class="container"
          >
            <div
              class="row justify-content-center"
            >
              <div
                class="col-md-8"
              >
                <div
                  class="h5 text-white pt-2"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
     
    <div
      class="page-content pb-2"
    >
      <div
        class="py-2 container-fluid"
        style="background-color: rgb(75, 46, 131);"
      >
        <div
          class="row justify-content-center"
        >
          <div
            class="col-md-8"
          >
            <img
              alt="UW Block Logo"
              class=""
              src="test-file-stub"
              width="40"
            />
          </div>
        </div>
      </div>
       
      <div
        class="container"
      >
        <div
          class="row justify-content-center"
        >
          <div
            class="page-content-padding col-md-12 col-lg-8"
          >
            <h1
              class="pt-md-5 pt-3 pb-2"
              id="standard-heading"
            >
              
    Welcome
  
            </h1>
             
            <div
              id="standard-body"
            >
              <p>
                Hungry Husky lets you share your leftover event food with those who
        need it most in the UW community. With your help, we can help reduce
        food insecurity among students and others—and reduce food waste at
        the same time.
              </p>
               
              <p>
                We’ll guide you through all the steps needed to share your food
        safely and responsibly.
              </p>
            </div>
             
            <div
              class="mt-5"
            >
              <div
                class="row justify-content-between"
              >
                <div
                  class="col-md-6 col-lg-5 order-md-2"
                >
                  <button
                    class="btn mb-3 btn-primary btn-lg btn-block"
                    style="white-space: nowrap;"
                    type="submit"
                  >
                    
                      Let's get started
          
                  </button>
                </div>
                 
                <div
                  class="col-md-6 col-lg-5 order-md-1"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
     
    <footer
      class="text-center"
      id="relative-footer"
    >
      <a
        href="mailto:help@uw.edu?subject=Hungry Husky support"
      >
        
        Contact support
      
      </a>
       
      <p>
        © 2019 University of Washington
      </p>
    </footer>
  </div>
</div>
`;
